spec/mailers/scheduled_publish_mailer_spec.rb

```diff
@@ -72,4 +72,24 @@
       )
     end
   end
+
+  describe "sending an email when scheduled edition has failed to published" do
```

Comment:
Minor, but I think it's more conventional to describe method names for unit tests like these e.g. `describe "#failure_email"`.